### PR TITLE
Make description easier to read

### PR DESCRIPTION
**Pull request by masesuraj112**
This commit edited the part of the text in the customising icons page and made it more clear to read. 

**Text that is unclear/ambiguous**

Original description I am modifying: "Let’s provide some actual icons. Download [this icons.zip bundle](https://docs.beeware.org/en/latest/_downloads/a8f92cddc5b6e0f777b9d9b223d31cce/icons.zip), and **unpack** it into the root of your project directory. **After unpacking, your project directory should look something like:**"

The problem was that I didn't understand what the term "unpack" means and I was concerned that users who encountered this word would be stuck on how to proceed. I also felt that there should be an additional steps on how to place the files into the root directory as I was initially unsure on how to do so when reading the instructions. I felt that there was ambiguity on which directory we should place the files in.

## PR Checklist:

- [ x] All new features have been tested
- [x ] All new features have been documented
- [ x] I have read the **CONTRIBUTING.md** file
- [ x] I will abide by the code of conduct

### CODE REVIEW

File: docs/tutorial/topics/custom-icons.rst [hunk 1]
```diff
@@ -161,9 +161,15 @@ This reports the specific icon file (or files) that Briefcase is expecting.
...
 Briefcase falls back to a default value (the same "gray bee" icon).
 
+
+
+
 Let's provide some actual icons. Download :download:`this icons.zip bundle
-<../resources/icons.zip>`, and unpack it into the root of your project
-directory. After unpacking, your project directory should look something like::
+<../resources/icons.zip>`, and unzip the file. To place the files in the root of your 
+helloworld directory, drag the icons.zip files manually to the directory in vsCode.
```
Comment (freakboy3742) [inline docs/tutorial/topics/custom-icons.rst:169]: This presumes you're using VSCode - which won't necessarily be true.

File: docs/tutorial/topics/custom-icons.rst [hunk 2]
```diff
@@ -161,9 +161,15 @@ This reports the specific icon file (or files) that Briefcase is expecting.
 However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
+
+
+
```
Comment (freakboy3742) [inline docs/tutorial/topics/custom-icons.rst:166]: Reduce, reuse, recycle your electrons! We don't need all these extra blank lines.

```suggestion
```

File: docs/tutorial/topics/custom-icons.rst [hunk 3]
```diff
@@ -161,9 +161,15 @@ This reports the specific icon file (or files) that Briefcase is expecting.
...
 Let's provide some actual icons. Download :download:`this icons.zip bundle
-<../resources/icons.zip>`, and unpack it into the root of your project
-directory. After unpacking, your project directory should look something like::
+<../resources/icons.zip>`, and unzip the file. To place the files in the root of your 
+helloworld directory, drag the icons.zip files manually to the directory in vsCode.
+Your project directory should look something like::
+
+
```
Comment (freakboy3742) [inline docs/tutorial/topics/custom-icons.rst:172]: Again, we don't need the extra newlines here.

```suggestion
```

**Review by freakboy3742 — CHANGES_REQUESTED** [review]
There's definitely something worth improving here, but the specific changes you've proposed here are problematic - first all the extra newlines, but second that it presumes the use of a specific editor.

File: docs/tutorial/topics/custom-icons.rst [hunk 4]
```diff
@@ -162,8 +162,8 @@ However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
 Let's provide some actual icons. Download :download:`this icons.zip bundle
-<../resources/icons.zip>`, and unpack it into the root of your project
-directory. After unpacking, your project directory should look something like::
+<../resources/icons.zip>`, and unzip the file. We can then place the icons
+folder into the root of the helloworld directory. Your project directory should look something like::
```
Comment (garthk) [inline docs/tutorial/topics/custom-icons.rst:166]: Double-backticks here would be consistent with earlier parts of the document, e.g. the advice on editing `pyproject.toml`:
```suggestion
<../resources/icons.zip>`, and unzip the file. We can then place the ``icons``
folder into the root of the ``helloworld`` directory. Your project directory should look something like::
```

File: docs/tutorial/topics/custom-icons.rst [hunk 5]
```diff
@@ -162,8 +162,8 @@ However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
 Let's provide some actual icons. Download :download:`this icons.zip bundle
```
Comment (garthk) [inline docs/tutorial/topics/custom-icons.rst:164]: Are we up for a `code-block:: console` here to show how to use `curl` or `wget` to download the bundle (Linux, macOS) and `unzip` to extract it? Would we do that as part of this PR, or a subsequent PR?

**Review by garthk — COMMENTED** [review]
Great start! I see you've already made some fixes; my apologies for any redundant feedback.

File: docs/tutorial/topics/custom-icons.rst (hunk 5 above)
Comment (masesuraj112) [reply docs/tutorial/topics/custom-icons.rst:164]: I've experimented with trying to add extra tables for new code-blocks yesterday and ran into many issues. I think this should be resolved in a subsequent PR.

File: docs/tutorial/topics/custom-icons.rst [hunk 6]
```diff
@@ -162,8 +162,8 @@ However, as we haven't provided the actual icon files, the install fails, and
...
-<../resources/icons.zip>`, and unpack it into the root of your project
-directory. After unpacking, your project directory should look something like::
+<../resources/icons.zip>`, and unzip the file. We can then place the ``icons``
+folder into the root of the ``helloworld`` directory. Your project directory should look something like::
```
Comment (freakboy3742) [inline docs/tutorial/topics/custom-icons.rst:166]: So - the problem I've got with this rewording is that starts with talking about an `icons` zip file, and then talks about an `icons` folder - and it isn't clear where that came from.

If we're not going to add the full explicit instructions for how to unzip the file, my preference would be to just replace the potentially ambiguous "unpack" with "unzip, and leave it at that:
```suggestion
<../resources/icons.zip>`, and unzip it into the root of your project
directory. After unzipping, your project directory should look something like::
```

(as an aside - we wrap at an 88 character maximum).

**Review by freakboy3742 — APPROVED** [review]
Looks good! Apologies for all the back-and-forth on this!

**Pull request merged**